project-base/docker/php-fpm/Dockerfile

```diff
@@ -48,8 +48,8 @@ RUN apk add --update nodejs-npm
 # install grunt-cli using npm to be able to run grunt watch
 RUN npm install -g grunt-cli
 
-# install postgresql to allow execution of pg_dump for acceptance tests (using older repository to install version 9.5)
-RUN apk add --update --no-cache --repository http://dl-cdn.alpinelinux.org/alpine/v3.4/main "postgresql<9.6"
+# install postgresql to allow execution of pg_dump for acceptance tests
+RUN apk add --update --no-cache --repository http://dl-cdn.alpinelinux.org/alpine/edge/main postgresql
```

Comment:
We do not prefer to use edge versions, I think that http://dl-cdn.alpinelinux.org/alpine/latest-stable/ would be preferable.